CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing
+
+You will find here the conventions and rules to publish new blog posts and to develop new blog features and enhancements.
+
+Please,
+- [open an issue](https://github.com/src-d/blog/issues) to request any new feature or to report a bug.
+- [create a new PR](https://github.com/src-d/blog/pulls) to request the review of a new post or feature.
+
+
+## Table of contents
+
+<!-- TOC -->
+
+- [Contributing](#contributing)
+    - [Table of contents](#table-of-contents)
+    - [Preview the blog posts](#preview-the-blog-posts)
+    - [Creating a new post](#creating-a-new-post)
+        - [Content schema](#content-schema)
+        - [Front matter](#front-matter)
+        - [Formatting the content](#formatting-the-content)
+        - [Links to other source{d} blog posts](#links-to-other-sourced-blog-posts)
+        - [Assets](#assets)
+        - [Authors](#authors)
+    - [Peer review](#peer-review)
+    - [How to publish blog posts and deploy the blog](#how-to-publish-blog-posts-and-deploy-the-blog)
+
+<!-- /TOC -->
+
+
+## Preview the blog posts
+
+While you are developing or creating new contents for the blog, and always before publishing a PR, you should validate your changes locally, specially if you're modifying common features or you're using shortcodes.
+
+To locally serve the blog, you need to satisfy the [project requirements](README.md#requirements), and then to run:
+
+```shell
+make serve;
+```
+and then, go to [http://localhost:8484](http://localhost:8484)
+
+
+## Creating a new post
+
+Posts are stored under [`post` content directory](content/post)
+
+A blog post content stored under: `content/post/__POST_FILE_NAME__.md`
+
+will be accessible under: `//__BLOG_HOST_NAME__/post/__POST_FILE_NAME__`
+
+
+### Content schema
+
+Every blog post content file is stored under the path `content/post/__POST_FILE_NAME__.md`, and has the following schema:
+
+```
+---
+author: authorKeyName
+date: 2006-01-02
+title: "Post title"
+image: /post/__POST_FILE_NAME__/header-image.png
+description: "Short description of the post"
+categories: ["science", "technical", "culture"]
+draft: true
+---
+
+Whatever content in `markdown` format.
+
+```
+
+
+### Front matter
+
+The **front matter** is the first section inside the content file &ndash;enclosed by "`---`" chars&ndash; where it is defined the content metadata.<br />(Read more about it in the Hugo docs for the [Front Matter section](https://gohugo.io/content-management/front-matter))
+
+- `author`: Defines the authorship as explained in the [authors section](#authors)
+- `date`: Date when you plan to publish the post, `2006-01-02` format. **Do not forget to update it before publishing it.**
+- `title`: Post title, as it will appear in the very top of the content
+- `image`: Defines the content listing image as explained in the [assets section](#assets)
```

Comment:
s/Defines the content listing image/Defines the image that will be shown in the list of posts/